tests/flow/tests_commands.py

```diff
@@ -536,3 +536,70 @@ def run_model_execute_from_llapi():
     info = info_to_dict(con.execute_command('AI.INFO', 'm{1}'))
     env.assertGreaterEqual(info['calls'], 0)
     env.assertGreaterEqual(num_parallel_clients, info['calls'])
+
+
+def test_ai_config(env):
+    if not TEST_PT:
+        env.debugPrint("skipping {} since TEST_PT=0".format(sys._getframe().f_code.co_name), force=True)
+        return
+    con = get_connection(env, '{1}')
+    model = load_file_content('pt-minimal.pt')
+
+    # Get the default configs.
+    res = con.execute_command('AI.CONFIG', 'GET', 'BACKENDSPATH')
+    env.assertEqual(res, None)
+    res = con.execute_command('AI.CONFIG', 'GET', 'MODEL_CHUNK_SIZE')
+    env.assertEqual(res, 511*1024*1024)
```

Comment:
let's validate those responses from every shard in the cluster, not just the one responsible for `{1}`